crates/ingress-http/src/handler/responses.rs

```diff
@@ -14,15 +14,18 @@ use bytes::Bytes;
 use http::{header, HeaderName, Response};
 use http_body_util::Full;
 use restate_schema_api::invocation_target::InvocationTargetMetadata;
+use restate_types::identifiers::InvocationId;
 use restate_types::ingress::IngressResponseResult;
 use restate_types::invocation::InvocationTarget;
 use tracing::{info, trace};
 
-const IDEMPOTENCY_EXPIRES: HeaderName = HeaderName::from_static("idempotency-expires");
+pub(crate) const IDEMPOTENCY_EXPIRES: HeaderName = HeaderName::from_static("idempotency-expires");
+pub(crate) const X_RESTATE_ID: HeaderName = HeaderName::from_static("x-restate-id");
```

Comment:
Maybe add a comment explaining what the expected value should be (stringified representation of InvocationId, I assume).